aptos-node/src/lib.rs

```diff
@@ -594,6 +600,8 @@ pub fn setup_environment_and_start_node(
     let (
         network_runtimes,
         consensus_network_interfaces,
+        dkg_network_interfaces,
+        jwk_consensus_network_interfaces,
```

Comment:
shall we wrap this under consensus instead of putting them at top level? @zekun000

Reply:
i believe zekun suggested this, lol